server/server.go

```diff
@@ -145,6 +145,7 @@ func (c *Server) setupRouter(ctx context.Context, logger *logrus.Logger) (contex
 	}
 
 	r.Mount("/v1/blindedToken", c.tokenRouter())
+	r.Mount("/v1/newBlindedToken", c.markedTokenRouter())
```

Comment:
I was thinking that we could just bump the API version here v2, so the path would be
```
/v2/blindedToken
```
are you thinking that
```
/v1/newBlindedToken
```
is more appropriate in this case?